pkg/operator/ceph/object/objectstore.go

```diff
@@ -358,3 +361,73 @@ func GetObjectBucketProvisioner(c *clusterd.Context, namespace string) string {
 	}
 	return provName
 }
+
+func checkDashboardUser(context *Context) bool {
+	args := []string{"dashboard", "get-rgw-api-access-key"}
+	cephCmd := ceph.NewCephCommand(context.Context, context.ClusterName, args)
+	out, _ := cephCmd.Run()
+	if string(out) == "" {
+		return false
+	} else {
```

Comment:
The `else` is not necessary, just `return true` at the end or change your condition with `!= ""` and return false by default.